docs/data/routes/docs/nextjs/getting-started-nextjs/why-nextjs/en.md

```diff
@@ -9,6 +9,10 @@ title: Why Next.js and JSS
 
 The latest JSS SDK leverages many of Next.js' [features](https://nextjs.org/#features), to simplify and enhance development workflow and experience. 
 
+<p>
+<iframe width="672" height="378" src="https://www.youtube.com/embed/CfoOKlKjzfA" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</p>
+
```

Comment:
Would it be useful to add a link to "more videos" somewhere?
https://www.youtube.com/playlist?list=PL1jJVFm_lGnwZup4L4BjITS2sKr4rpMfI